tests/e2e/vsphere_volume_expansion.go

```diff
@@ -1495,12 +1512,12 @@ var _ = ginkgo.Describe("Volume Expansion Test", func() {
 			err = fmt.Errorf("queryCNSVolumeWithResult returned no volume")
 		}
 		gomega.Expect(err).NotTo(gomega.HaveOccurred())
+
 		ginkgo.By("Verifying disk size requested in volume expansion is honored")
 		newSizeInMb := int64(3072)
 		if queryResult.Volumes[0].BackingObjectDetails.(*cnstypes.CnsBlockBackingDetails).CapacityInMb != newSizeInMb {
 			err = fmt.Errorf("got wrong disk size after volume expansion")
 		}
-		gomega.Expect(err).NotTo(gomega.HaveOccurred())
```

Comment:
we are creating an error for wrong disk size after volume expansion above, we should keep the check `gomega.Expect(err).NotTo(gomega.HaveOccurred())`